.gitignore

```diff
@@ -27,7 +27,12 @@ tools/Cake
 packages/
 # !packages/repositories.config
 
+# NUnit results
+TestResult.xml
 
+# Tools installed by Cake
+tools/
+!tools/packages.config
```

Comment:
Agreed it's good to control the version - in that case do you perhaps need to force add the packages.config to git? It doesn't seem to have got into the PR. :-)